dist/js/script.js

```diff
@@ -315,7 +367,8 @@
       const thisCart = this;
       thisCart.dom = {};
       thisCart.dom.wrapper = element;
-      thisCart.dom.toggleTrigger = thisCart.element.querySelector(select.cart.toggleTrigger);
+      thisCart.dom.toggleTrigger = thisCart.dom.wrapper.querySelector(select.cart.toggleTrigger);
+      thisCart.dom.productList = thisCart.dom.wrapper.querySelector(select.cartProduct.productList);
```

Comment:
Tutaj chyba powinno być `select.cart.productList`. W obiekcie `cartProduct` nie ma właściwości `productList` ;) Porównaj w linijkach `49-54` 
```suggestion
      thisCart.dom.productList = thisCart.dom.wrapper.querySelector(select.cart.productList);
```